src/components/themr.js

```diff
@@ -96,9 +96,8 @@ export default (componentName, localTheme, options = {}) => (ThemedComponent) =>
 
 export function themeable(style = {}, theme) {
   if (!theme) return style
-  return Object.keys(theme).reduce((result, key) => (
-    Object.assign(result, { [key]: `${style[key]} ${theme[key]}` })
-  ), Object.assign({}, style))
+  return Object.keys(theme).reduce((result, key) =>
+    ({ ...result, [key]: `${style[key]} ${theme[key]}` }), style)
```

Comment:
Guys, after recent changes here this line now produces `undefined` in `${style[key]}` interpolation if `style` does not containg a key from `theme`.